### PR TITLE
Basic functionality and styling

### PR DESCRIPTION
**Pull request by kevinwchiang**

### CODE REVIEW

File: src/components/Charts.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          <Circle percent={inProgressPercentage} strokeWidth="6" strokeColor={'#33ff33'} />
+        </div>
+        <div style={circleContainerStyle}>
+          <div style={counterStyle}>
+            <span style={counterNumStyle}>{tasks.complete}</span> Complete
+            <p>{completePercentage}%</p>
+          </div>
+          <Circle percent={completePercentage} strokeWidth="6" strokeColor={'#33ff33'} />
+        </div>
+      </div>
```
Comment (alexpeachey) [inline src/components/Charts.js:24]: The code for each chart looks strikingly similar. I might extract to a function or even a single chart component.

File: src/reducers/projects.js [hunk 2]
```diff
@@ -40,6 +49,8 @@ const tasks = (state, action) => {
       ])
     case START_TASK:
       return calculateCounters(state.list.map(t => task(t, action)))
+    case COMPLETE_TASK:
+      return calculateCounters(state.list.map(t => task(t, action)))
```
Comment (alexpeachey) [inline src/reducers/projects.js:52]: looks like the code for both `START_TASK` and `COMPLETE_TASK` are the same so the cases could be collapsed

File: src/reducers/projects.js [hunk 3]
```diff
@@ -76,6 +95,8 @@ const projects = (state = [], action) => {
     case ADD_TASK:
     case START_TASK:
       return state.map(p => project(p, action))
+    case COMPLETE_TASK:
+      return state.map(p => project(p, action))
```
Comment (alexpeachey) [inline src/reducers/projects.js:91]: Same down here.

**Comment by alexpeachey** [comment]
Overall fairly clean looking code. I had a few comments that could tighten up some of the duplication.

**Comment by kevinwchiang** [comment]
@alexpeachey Thanks for the suggestions, and agreed. I may have some extra time away from work tonight or over the weekend if you wanted me to work on writing some tests for the code I wrote as well. Please let me know if you'd like to see some tests!

**Pull request closed without merging**